### PR TITLE
QOL-6491 Enable proxy for resource downloads

### PR DESCRIPTION
**Pull request by ThrawnCA**
- Use proxy server for resource downloads if configured
- Don't override the Archiver and Report versions, just ensure they're installed
- Cleanup to prepare for Python 3

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ messytables==0.15.2
 progressbar==2.3
 #SQLAlchemy>=0.6.6 #in ckancore
 #requests==2.11.1 #in ckancore
-#six==1.11.0 #in ckancore
+six>=1.0.0 #in ckancore
 
--e git+https://github.com/qld-gov-au/ckanext-archiver.git@2.1.0-qgov.1#egg=ckanext-archiver
--e git+https://github.com/qld-gov-au/ckanext-report.git@0.1#egg=ckanext-report
+ckanext-archiver
```
Comment (duttonw) [inline requirements.txt:9]: can you add it to dev requirements or does upstream have our changes now?

Comment (ThrawnCA) [reply requirements.txt:9]: The README for this extension says to install archiver and report first (which makes sense, as they're installed from source not from pypi). So the requirements file shouldn't install them, just verify.

Comment (duttonw) [reply requirements.txt:9]: do add, for cicd integration testing to work. unless there is some other script already added it.

Comment (ThrawnCA) [reply requirements.txt:9]: The Travis config already installs them manually.

**Review by duttonw — APPROVED** [review]

**Pull request merged**